backend/lib/services/shoots.js

```diff
@@ -372,14 +364,10 @@ exports.info = async function ({ user, namespace, name }) {
   }
   data.dashboardUrlPath = getDashboardUrlPath(shoot.spec.kubernetes.version)
 
-  /*
-    We explicitly use the (privileged) dashboardClient here for fetching the monitoring credentials instead of using the user's token
-    as we agreed that also project viewers should be able to see the monitoring credentials.
-    Usually project viewers do not have the permission to read the <shootName>.monitoring credential.
-    Our assumption: if the user can read the shoot resource, the user can be considered as project viewer.
-    This is only a temporary workaround until a Plutono SSO solution is implemented https://github.com/gardener/monitoring/issues/11.
-  */
-  await assignMonitoringSecret(dashboardClient, data, namespace, name)
+  const oidcObservabilityUrlsEnabled = _.get(config, 'frontend.features.oidcObservabilityUrlsEnabled', false)
+  if (!oidcObservabilityUrlsEnabled && await authorization.canGetSecret(user, namespace, `${name}.monitoring`)) {
+    await assignMonitoringSecret(client, data, namespace, name)
+  }
```

Comment:
```suggestion
  const oidcObservabilityUrlsEnabled = _.get(config, 'frontend.features.oidcObservabilityUrlsEnabled', false)
  if (!oidcObservabilityUrlsEnabled) {
    const canGetMonitoringSecret = await authorization.canGetSecret(user, namespace, `${name}.monitoring`)
    if (canGetMonitoringSecret) {
      await assignMonitoringSecret(client, data, namespace, name)  
    } 
  }
```

Reply:
Why NOT enabled (`!oidcObservabilityUrlsEnabled`)?

Reply:
When the oidc observability URLs feature is not enabled, we need username/password to access the UIs using the basic auth ingress URLs.
If enabled, we do not need the username and password